public/locales/en/translation.json

```diff
@@ -187,26 +187,22 @@
   },
   "retrocert-login": {
     "title": "Retroactive Certification for Unemployment",
-    "subheader": "Help us find you",
-    "help-text": "We need to make sure your identity matches EDD records so that we show you accurate records.",
-    "last-name-label": "Last name",
-    "dob-heading": "Date of Birth",
+    "help": "You are required to certify for Unemployment Insurance (UI) and Pandemic Unemployment Assistance (PUA) benefits from the week ending February 9, 2020, through week ending May 9, 2020.",
```

Comment:
content comments that might be out of the scope of this PR
1) based on the data and for consistency with May 9, should be the week ending February 8th
2) should be "through the week" grammatically

Reply:
Interesting.... I'll bring this up in the document.